venthur_api/libardrone.py

```diff
@@ -130,6 +130,16 @@ def set_speed(self, speed):
         """
         self.speed = speed
 
+    def increase_speed(self):
+        """Increase the speed by an increment of 0.1."""
+        if (self.speed + 0.1) <= 1:
+            self.speed += 0.1
+
```

Comment:
You could also do:

self.speed = min((1, self.speed + 0.1))

to do the same thing on a single line.